code/modules/projectiles/guns/mounted.dm

```diff
@@ -1,12 +1,11 @@
 /obj/item/weapon/gun/energy/gun/advtaser/mounted
 	name = "mounted taser"
 	desc = "An arm mounted dual-mode weapon that fires electrodes and disabler shots."
-	icon_state = "armcannon"
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "taser"
+	item_state = "armcannonstun4"
```

Comment:
These sprites are made by Randy and slightly modified by me. The first one was made for a cyberpunkish build that eventually became Lifeweb, the second one is a badass resprite of original (very old) egun.

Inhands are still old ones. They don't update charge and don't turn invisible anymore. Maybe I'll change them.